concurrent/src/main/scala/scalaz/concurrent/Task.scala

```diff
@@ -247,15 +247,15 @@ object Task {
           a.map((_, residuals.map(new Task(_))))
         })
       val AE = Apply[Throwable \/ ?]
-      override def reduceUnordered[A, M](fs: Seq[Task[A]])(implicit R: Reducer[A, M]): Task[M] = {
-        import R.monoid
-        val RR: Reducer[Throwable \/ A, Throwable \/ M] =
+      override def reduceUnordered[A, M](fs: Seq[Task[A]])(implicit R: Reducer[A, M], M: Monoid[M]): Task[M] = {
+        implicit val MM: Monoid[Throwable \/ M] = Monoid.liftMonoid[Throwable \/ ?, M]
```

Comment:
Inserting a `Monoid` in the middle of code seems wonky. A `Monoid` is a type-class right?

Reply:
yes, same thing for the reducer above.

Reply:
Created #1628 to address the issue separately and at large (as this PR did not introduced the local instances).